pages/about/participating.md

```diff
@@ -39,121 +35,81 @@ This page explains how you can participate in WAI work, for example:
-    managers, and others. Use the [curricula](https://www.w3.org/WAI/curricula/) to develop courses.
--   Look around the W3C WAI website to find other [accessibility resources](https://www.w3.org/WAI/resources/) for specific [roles](https://www.w3.org/WAI/roles/) that you want to share on social media, within your organization, and with others.
-
-### Implementing Guidelines
-
--   Use the [Web Content Accessibility Guidelines
-    (WCAG)](/standards-guidelines/wcag/) documents to help make your website
-    accessible.
--   If you develop authoring tools—any software or service that people
-    use to create or modify web content, including content management
-    systems—implement the [Authoring Tool Accessibility Guidelines
-    (ATAG)](/standards-guidelines/atag/).
--   If you develop web browsers, media players, assistive technologies,
-    or other user agents—implement the [User Agent Accessibility
-    Guidelines (UAAG)](/standards-guidelines/uaag/).
-
-### Encouraging Accessibility
-
--   [Encourage authoring tools](/WAI/impl/software) to meet
-    [ATAG](/standards-guidelines/atag/) by directly contacting
-    vendors and requesting increased accessibility support in future
-    versions, and by purchasing tools that provide the best support for
-    accessibility.
--   Encourage web browsers, media players, assistive technologies, and
-    other user agents to meet
-    [UAAG](/standards-guidelines/uaag/).
--   Encourage organizations to [adopt an accessibility
-    policy](/WAI/impl/pol) for their organization that defines their
-    commitment to web accessibility.
--   Encourage websites to be accessible, for example, provide [feedback
-    on inaccessible websites and inaccessible web products
-    carefully](http://www.w3.org/WAI/users/inaccessible.html); it is
-    usually more productive to start with a positive encouraging tone,
-    rather than a negative critical tone.
+To be notified about draft documents for review, see [Subscribe to WAI News](https://www.w3.org/WAI/news/subscribe/).
+
+## Join a community group
+
+Participate in a community group and collaborate with others to improve digital accessibility &mdash; everyone is welcome to join. To find a list of active community groups hosted by W3C that focus on digital accessibility issues, search for ‘accessibility’ on [Current Groups — W3C Community and Business Groups](https://www.w3.org/community/groups).
+
+These are some community groups that are currently seeking contributors:
+
+- [ACT Rules Community Group](https://www.w3.org/community/act-r/)
+- [ARIA and Assistive Technologies Community Group](https://www.w3.org/community/aria-at/)
+- [Cognitive Accessibility Community Group](https://www.w3.org/community/coga-community/)
+- [Immersive Captions Community Group](https://www.w3.org/community/immersive-captions/)
+
+Details for joining are on the respective community group pages.
+
+## Join a working group
```

Comment:
This and 'Join a TF' I find a bit difficult. This relates to the discussion on IEs but also covers how membership works.

I am slightly conflicted about linking through to Membership information though.

@shawna-slh Would welcome your thoughts - or a chat about this one.